### DISCUSSION TITLE
Strange symbols on logout

### DISCUSSION THREAD

**Discussion by Bluey26** [lxqt/lxqt#2617] [Q&A]
Hello, i have been experiencing the following "issue" from some time already.

When i logout from the LXQt session, using the menu>Leave button>Logout , from time to time, i see the following characters in the logout screen:

`^@^@^@^@^@^@^@^@^@^@^@^@@^` (The "normal" logout message usually is `_`)

The amount of those is random, and sometimes they do not appear at all. The amount usually increases with the uptime(time since the power on).

I thought this could be related to the WM which i use(Labwc) but a few minutes ago i realized that the problem or issue casuing this may be somewhere else, since i logged-in into X11 to check a thing(only a few minutes), and in the logout scene i saw the same message.

Since the logout is made using `lxqt-leave --logout`(this is what is executed by the logout button, i guess), i checked to what LXQt package it did belong.

lxqt-leave seems to belong to `lxqt-session` according to https://man.archlinux.org/man/lxqt-leave.1

I really do not know what that causes, it seems to be a harmless message but to be in the safe side i decided to report it in here. I mentioned it to stef some time ago but since the debugging of this does not seem to be easy, i did not take more measures to see where could be the issue.

From what i recall this used to not happen to me before all the wayland steps made in the past time(unfortunately i dont remember when this started exactly).

Another thing that may be causing it would be some kind of program that is not finished "gracefully" before running `lxqt-leave --logout` and that strange code could be some kind of error or message from that program.

As i said, this does not seem to be a problem and seems hard to debug, but i wanted to comment it in here since i saw it was not related to the WM i was using in Wayland.

**Comment by marcusbritanicus** [comment]
I do not think this is related to LXQt at all. I have noticed it when using Wayfire and Sway and just killing the wm (`pkill wayfire` or `pkill sway`), while not using LXQt or it's components.

**Reply by Bluey26** [reply]
Could it be some program which is not closing properly or maybe some kind of package which is causing the problem?

Now i really have no idea about this, but it seems harmless

**Reply by tsujan** [reply]
> Could it be some program which is not closing properly

As far as I know, there's no correlation between the two.

I haven't encountered it here (with kwin_x11, kwin_wayland, labwc and wayfire), but my wild guess is that it may be related to the display manager (SDDM?).

**Reply by marcusbritanicus** [reply]
I second what you say. It's very unlikely that some user app does this. But, it might be more to do with (e)logind.  I do no use sddm and yet see those artefacts. Also, only a very few programs will have access to the underlying raw terminal.
